### PR TITLE
Feature/9873 extend local statistics

### PR DESCRIPTION
**Pull request by naveeddotio**
## Description
Changes to extend local statistics to support hospitalisation rate as a secondary field.

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9873

## Screenshots
Simulator Screen Shot - iPhone 12 - 2021-10-12 at 16 09 23
Simulator Screen Shot - iPhone 12 - 2021-10-12 at 16 09 30
Simulator Screen Shot - iPhone 12 - 2021-10-12 at 16 09 35
Simulator Screen Shot - iPhone 12 - 2021-10-12 at 16 09 18

### CODE REVIEW

**Review by KathrinLuecke — APPROVED** [review]
UA approved

File: src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCardView/HomeStatisticsCardViewModel.swift [hunk 1]
```diff
@@ -77,8 +77,26 @@ class HomeStatisticsCardViewModel {
...
+			secondaryTrendImage = sevenDayHospitalizationTrend.trendImage
+			secondaryTrendImageTintColor = sevenDayHospitalizationTrend.trendTintColor
+			secondaryTrendAccessibilityLabel = sevenDayHospitalizationTrend.trendAccessibilityLabel
+			secondaryTrendAccessibilityValue = sevenDayHospitalizationTrend.trendAccessibilityValue
+
+			let sevenDayHospitalizationIncidenceUpdatedDate = Date(timeIntervalSince1970: TimeInterval(sevenDayHospitalizationIncidenceUpdatedAt))
+			secondaryTitle = sevenDayHospitalizationIncidenceUpdatedDate.formatted(
+				todayString: AppStrings.Statistics.Card.Combined7DaysIncidence.today,
+				yesterdayString: AppStrings.Statistics.Card.Combined7DaysIncidence.yesterday,
+				otherDateString: AppStrings.Statistics.Card.Combined7DaysIncidence.date
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCardView/HomeStatisticsCardViewModel.swift:93]:
```suggestion
				todayString: AppStrings.Statistics.Card.Region.today,
				yesterdayString: AppStrings.Statistics.Card.Region.yesterday,
				otherDateString: AppStrings.Statistics.Card.Region.date
```

I think it makes sense to use the new strings that Omar introduced in his PR :)

**Review by ArturFriesen — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCardView/HomeStatisticsCardViewModel.swift [hunk 2]
```diff
@@ -77,8 +77,26 @@ class HomeStatisticsCardViewModel {
 		} else {
 			primaryTitle = String(format: AppStrings.Statistics.Card.Incidence.date, "")
 		}
-
 		primarySubtitle = AppStrings.Statistics.AddCard.localCardPrimarySubtitle
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCardView/HomeStatisticsCardViewModel.swift:80]: I think this one is outdated and should just say "Neuinfektionen" now

File: src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCardView/HomeStatisticsCardViewModel.swift [hunk 3]
```diff
@@ -77,8 +77,26 @@ class HomeStatisticsCardViewModel {
...
+			let sevenDayHospitalizationIncidenceUpdatedDate = Date(timeIntervalSince1970: TimeInterval(sevenDayHospitalizationIncidenceUpdatedAt))
+			secondaryTitle = sevenDayHospitalizationIncidenceUpdatedDate.formatted(
+				todayString: AppStrings.Statistics.Card.Combined7DaysIncidence.today,
+				yesterdayString: AppStrings.Statistics.Card.Combined7DaysIncidence.yesterday,
+				otherDateString: AppStrings.Statistics.Card.Combined7DaysIncidence.date
+			)
+		} else {
+			secondaryTitle = String(format: AppStrings.Statistics.Card.Incidence.date, "")
+		}
+		secondarySubtitle = AppStrings.Statistics.Card.Combined7DaysIncidence.secondaryLabelSubtitle
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCardView/HomeStatisticsCardViewModel.swift:98]: This one must differentiate between local statistics for a federal state or administrative unit and for an administrative unit it should say "Hospitalisierungen in {federal state name}" (See Figma)

**Review by nickguendling — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCardView/HomeStatisticsCardViewModel.swift (hunk 2 above)
Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCardView/HomeStatisticsCardViewModel.swift]: Fixed the text

**Review by nickguendling — APPROVED** [review]

**Pull request merged**